installer/x86_64/install.sh

```diff
@@ -55,7 +53,17 @@ ONIE_PLATFORM_EXTRA_CMDLINE_LINUX=""
 # Default var/log device size in MB
 VAR_LOG_SIZE=4096
 
-[ -r platforms/$onie_platform ] && source platforms/$onie_platform
+
+# simple detect whether script was run in SONiC
```

Comment:
You should reuse the code to detect ONIE/Sonic.

---
In reply to: [109438459](https://github.com/Azure/sonic-buildimage/pull/464#discussion_r109438459) [](ancestors = 109438459)